src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -324,9 +318,31 @@ public WebView(WebView webView) : this(Interop.WebView.NewWebView(WebView.getCPt
         [EditorBrowsable(EditorBrowsableState.Never)]
         public WebSettings Settings { get; }
 
+        /// <summary>
+        /// The uri to load.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Uri Uri
+        {
+            get
+            {
+                return new Uri((string)GetValue(UrlProperty));
+            }
+            set
+            {
+                if (value != null)
+                {
+                    SetValue(UrlProperty, value.AbsoluteUri);
+                    NotifyPropertyChanged();
+                }
+            }
+        }
+
         /// <summary>
         /// The url to load.
         /// </summary>
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Please remove `since_tizen` tag for hidden APIs.

Reply:
Done.